ibm_was/datadog_checks/ibm_was/data/conf.yaml.example

```diff
@@ -8,31 +8,93 @@ instances:
     #
   - servlet_url: http://localhost:<PORT>/wasPerfTool/servlet/perfservlet
 
-    ## @param skip_proxy - boolean - optional - default: False
+    ## @param username - string - optional
+    ## The username to use for authentication againse the servlet
+    #
+    # username: <USERNAME>
+
+    ## @param password - string - optional
+    ## The username to use for authentication againse the servlet
+    #
+    # password: <PASSWORD>
+
+    ## @param timeout - integer - optional - default: 10
+    ## The timeout for connecting to the perf servlet
+    #
+    # timeout: 10
+
+    ## @param headers - object - optional
+    ## Map of headers to send on every request
+    #
+    # headers:
+    #   <KEY_1>: <VALUE_1>
+    #   <KEY_2>: <VALUE_2>
+
+    ## @param tls_veify - boolean - optional - default: true
+    ## Whether or not to verify TLS certificates.
+    #
+    # tls_verify: true
+
+    ## @param tls_ca_cert - string - optional
+    ## The path to a CA_BUNDLE file or directory with certificates of trusted CAs. Implicitly sets tls_verify to true
+    #
+    # tls_ca_cert: <PATH_TO_CERT>
+
+    ## @param tls_cert - string - optional
+    ## The client side certificate to use
+    #
+    # tls_cert: <PATH_TO_CERT>
+
+    ## @param tls_private_key - string - optional
+    ## The private key to use if it's not included in the tls_cert file
+    #
+    # tls_private_key: <private_key>
+
+    ## @param tls_ignore_warning - boolean - optional - default: false
+    ## Whether or not to suppress insecure request warnings when tls_verify is false
+    #
+    # tls_ignore_warning: false
+
+    ## @param proxy - object - optional
+    ## Proxy configuration
+    #
+    # proxy:
+    #   http: <HTTP_PROXY>
+    #   https: <HTTPS_PROXY>
+    #   no_proxy:
+    #     - <HOSTNAME_1>
+    #     - <HOSTNAME_2>
+
+    ## @param skip_proxy - boolean - optional - default: false
     ## Wheter or not to skip the proxy configured in the Agent's config file when
     ## making the request to the PerfServlet
     #
-    # skip_proxy: False
+    # skip_proxy: false
+
+    ## @param persist_connections - boolean - optional - default: false
+    ## Whether or not to persist cookies and use connection pooling
+    #
+    # persist_connections: false
 
-    ## @param collect_jdbc_stats - boolean - optional - default: True
```

Comment:
Consistency aside does capitalisation matters?

Reply:
No, but I'll fix. Thanks!

Reply:
Yes, as yaml bools are lowercase. We do more value acceptance in https://github.com/DataDog/integrations-core/blob/7e794127f1312d6830880f35bee54ae2668f25f6/datadog_checks_base/datadog_checks/base/config.py#L6-L16